user_docs/en/changes.t2t

```diff
@@ -44,6 +44,7 @@ What's New in NVDA
   - The volume of the other applications can be adjusted by ``NVDA+alt+pageUp`` and ``NVDA+alt+pageDown``. (#16052, @mltony)
   - The sound of the other applications can be muted with ``NVDA+alt+delete``. (#16052, @mltony)
   -
+- In LibreOffice Writer (version 24.8 and newer), when toggling text formatting (bold, italic, underline, subscript/superscript, alignment) using the corresponding keyboard shortcut, NVDA announces the new formatting attribute (e.g. "Bold on", "Bold off"). (#4248, @michaelweghorn)

```

Comment:
this will need to be moved to 2024.3 when the change log section is created

Reply:
Is there anything to do for me at the moment or just wait for now?

Reply:
just wait